Brain_I2C/Brain_I2C_Basic.ino

```diff
@@ -0,0 +1,26 @@
+#include <Wire.h>
+
+void setup() {
+  Wire.begin();        // join i2c bus (address optional for master)
+  Serial.begin(9600);  // start serial for output
+}
+
+void loop() {
+  int sensor_id = 0; //change to desired sensor_id
+  Wire.requestFrom(8, 4); // request 4 bytes from address 8
+  uint8_t index = 0;
+  union floatToBytes {
```

Comment:
move outside of loop. Define it globally.
Currently, it wastes a lot of memory recreating this object, when we can just modify the same one each time.